package.json

```diff
@@ -156,11 +166,8 @@
   "browser": {
     "./index.js": "./browser-entry.js",
     "./lib/nodejs/growl.js": "./lib/browser/growl.js",
-    "tty": "./lib/browser/tty.js",
-    "./lib/cli/*.js": false,
-    "chokidar": false,
+    "./lib/esm-utils.js": false,
```

Comment:
so this browser field... I'm thinking it should just be `{"browser": "mocha.js"}`.  all of these mappings are for browserify.

Reply:
The rollup plugins use this field the same way browserify did. Removing these confiurations increases the bundle size by 200-300kb. So these remappings for a browser build are neede. But I'll see about getting them moved into the relevant rollup pllugins configuration instead so it becomes more obvious

Reply:
Fixed in 207ef63b